ar4_gazebo/launch/gazebo_moveit.launch.py

```diff
@@ -0,0 +1,101 @@
+#    and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""launch file for integrating Gazebo with MoveIt for the AR4 robot."""
+
+import launch
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription, DeclareLaunchArgument
+from launch.actions import GroupAction
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.substitutions import FindPackageShare
+from launch.substitutions import LaunchConfiguration
+import os
+
+
+def generate_launch_description():
+    """Launch the AR4 robot in Gazebo and MoveIt."""
+    ar4_gazebo_pkg = 'ar4_gazebo'
+    ar4_moveit_config_pkg = 'ar4_moveit_config'
+
+    # Declare launch argument for enabling/disabling the ros bridge
+    use_ros_bridge = LaunchConfiguration('use_ros_bridge')
+
+    declare_use_ros_bridge = DeclareLaunchArgument(
+        'use_ros_bridge', default_value='true', description='Enable ROS bridge'
+    )
+
+    # Launch ar4_in_empty_world.launch.py
+    gazebo_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                os.path.join(
+                    FindPackageShare(ar4_gazebo_pkg).find(ar4_gazebo_pkg),
+                    'launch',
+                    'ar4_in_empty_world.launch.py',
+                )
+            ]
+        )
+    )
+
+    # Launch gz_ros_bridge.launch.py only if use_ros_bridge is true
+    gz_ros_bridge_launch = GroupAction(
+        [
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(
+                    [
+                        os.path.join(
+                            FindPackageShare(ar4_gazebo_pkg).find(ar4_gazebo_pkg),
+                            'launch',
+                            'gz_ros_bridge.launch.py',
+                        )
+                    ]
+                )
+            )
+        ],
+        condition=launch.conditions.IfCondition(use_ros_bridge),
```

Comment:
Import `launch.conditions.IfCondition` so that you can use `IfCondition` directly.
```suggestion
        condition=IfCondition(use_ros_bridge),
```

Reply:
sure, thx